src/main/index.ts

```diff
@@ -74,6 +74,10 @@ function init() {
 
         registerScreenShareHandler();
         registerMediaPermissionsHandler();
+        //register file handler so we can load the custom splash animation from the user's filesystem
+        protocol.handle("splash-animation", () => {
+            return net.fetch("file:///"+splashAnimationPath);
```

Comment:
From the other PR, it would still be better if this was just a `fs.readSync` call

Reply:
I disagree, since the Electron protocol handler API [expects a promise for a fetch response](https://www.electronjs.org/docs/latest/api/protocol#protocolhandlescheme-handler) as the return value. 

Using a synchronous API for this isn't ideal because it will block the rest of the main process which will increase the startup time as it waits for the file to load.

The Electron docs also say to use `net.fetch` for reading local files: https://www.electronjs.org/docs/latest/breaking-changes#deprecated-protocolregisterinterceptbufferstringstreamfilehttpprotocol

Reply:
I was confusing it with interceptBufferProtocol the entire time then.